### DISCUSSION TITLE
봇 페이지의 서버 수를 버튼이 아닌 다른 요소 (div 등...)로 변경해 주세요.

### DISCUSSION THREAD

**Discussion by injoon5** [koreanbots/koreanbots#344] [일반] [answered]
현재 한디리의 모든 봇 상세 페이지에 들어가면 서버 수가 나옵니다. 여기에서 서버 수에 마우스를 올리면 마치 클릭이 될 것 같이 마우스 아이콘 모양이 바뀌는데요, 이는 button element 를 사용해서 인것 같습니다. 이로 인하여 사용자들이 클릭이 되는 것 같아 클릭을 해봐도 아무 일도 일어나지 않아 혼란을 빚을 수 있기 때문에, div 같은 다른 element 로 변경하거나, 호버를 해도 마우스 아이콘 모양이 바뀌지 않게 설정하는 것을 제안 드립니다.

**Comment by injoon5** [comment]
혹시라도 v2에서 변경된다면 그냥 덮어 주시면 됩니다...

**Comment by wonderlandpark** [comment, marked as answer]
1. Button element 여도 hover시 cursor 모양이 변경되지 않도록 하면됩니다.
2. v1에는 더 이상 패치되지 않습니다 (#125)

검사합니다
